src/Nethermind/Nethermind.Consensus.AuRa/Validators/ContractBasedValidator.cs

```diff
@@ -240,14 +240,21 @@ private void InitiateChange(Block block, Address[] potentialValidators, bool isP
 
         private Address[] LoadValidatorsFromContract(BlockHeader parentHeader)
         {
-            var validators = ValidatorContract.GetValidators(parentHeader);
+            try
+            {
+                var validators = ValidatorContract.GetValidators(parentHeader);
+                
+                if (validators.Length == 0)
+                {
+                    throw new AuRaException("Failed to initialize validators list.");
```

Comment:
you change AbiException to AuRa and  you changed the exception that you catch from AuRa to Abi. Was it intended?